client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -19,14 +19,25 @@ const leaveInvestigationMessage = 'צא מחקירה';
 const InvestigatedPersonInfo = (props: Props) => {
 
     const classes = useStyles();
-    const { investigatedPatientByInvestigatedPatientId, epedemioligyNumber, onExitInvestigation } = props;
+    const { currentTab, investigatedPatientByInvestigatedPatientId, epedemioligyNumber } = props;
 
     const Divider = () => <span className={classes.divider}> | </span>;
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
     const cantReachInvestigated = useSelector<StoreStateType, boolean>(state => state.investigation.cantReachInvestigated);
 
-    const { confirmExitUnfinishedInvestigation, handleCantReachInvestigatedCheck, getPersonAge } = useInvestigatedPersonInfo({onExitInvestigation});
+    const { confirmExitUnfinishedInvestigation, handleCantReachInvestigatedCheck, getPersonAge } = useInvestigatedPersonInfo();
+    
+    const handleLeaveInvestigationClick = (e: any) => {
+        e.preventDefault();
+        if(isEventTrigeredByMouseClicking(e)) {
```

Comment:
What happens if he leaves via chrome's back button? or via url?

Reply:
for now, nothing